SCons/Defaults.xml

```diff
@@ -30,12 +30,13 @@ A function used to produce variables like &cv-link-_CPPINCFLAGS;. It takes
 four or five
 arguments: a prefix to concatenate onto each element, a list of
 elements, a suffix to concatenate onto each element, an environment
-for variable interpolation, and an optional function that will be
-called to transform the list before concatenation.
+for variable interpolation, an optional function that will be
+called to transform the list before concatenation, affect_signature which will wrap non-empty returned value with
+    $( and $) to indicate the contents should not affect the signature of the generated command line.
 </para>
 
 <example_commands>
-env['_CPPINCFLAGS'] = '$( ${_concat(INCPREFIX, CPPPATH, INCSUFFIX, __env__, RDirs)} $)',
+env['_CPPINCFLAGS'] = '${_concat(INCPREFIX, CPPPATH, INCSUFFIX, __env__, RDirs, affect_signature=False)}',
```

Comment:
the trailing comma in the original looks like an error, we should not keep.

Reply:
Fixed in next push.